### ISSUE TITLE
Pretrained model Coco with Resnet50 layers

### ISSUE THREAD

**Issue by evinpinar** [multimodallearning/pytorch-mask-rcnn#93]
Currently, there are two pretrained model weights:
1. Resnet50 Imagenet weights for feature extraction (backbone)
2. MaskRcnn Coco weights for mask-rcnn with resnet101 architecture

Are there any coco trained maskrcnn weights for resnet50 architecture? Or is it okay if I use the resnet101model weights on resnet50 architecture?

**Comment by evinpinar** [comment]
So far, according to my experiments, it does not work well.

**Comment by bpmsilva** [comment]
@evinpinar, you could use [this tutorial](https://pytorch.org/tutorials/intermediate/torchvision_tutorial.html), which uses a native implementation of Mask RCNN in Pytorch

**Comment by evinpinar** [comment]
Thanks, but I need the weights because i'm modifying this implementation a bit. 
I guess i will instead move to fb official implementation.
